src/ChatWindow.js

```diff
@@ -13,43 +13,133 @@ import { users, userIds } from "./ConversingUsers";
 import AVSGateway from "./AVSGateway";
 const avs = new AVSGateway();
 
+const USER_ID = userIds.YOU;
```

Comment:
If YOU is the USER, change the enum to USER too. Also, since you are accessing the enum anyway, having a constant to represent this doesnt make sense.

Reply:
Changed to use the enum instead of assigning to another const. 

I will address changing the 'YOU' to something else in a different commit. Users.YOU sounds confusing but Users.USER also sounds confusing. Need to think about it and its not directly related to this commit anyways.

Reply:
Okay.